### PR TITLE
RHDEVDOCS-4872 - Logging 5.6.1 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.11+

Issue: [RHDEVDOCS-4872](https://issues.redhat.com//browse/RHDEVDOCS-4872)

Link to docs preview: https://55635--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-6-1_cluster-logging-release-notes-v5x

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-rn-5.6.1.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+* Before this update, the LokiStack Gateway Labels Enforcer generated parsing errors for valid LogQL queries when using combined label filters with boolean expressions. With this update, the LokiStack LogQL implementation supports label filters with boolean expression and resolves the issue. (link:https://issues.redhat.com/browse/LOG-3441[LOG-3441]), (link:https://issues.redhat.com/browse/LOG-3397[LOG-3397])
+
+* Before this update, records written to Elasticsearch would fail if multiple label keys had the same prefix and some keys included dots. With this update, underscores replace dots in label keys, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3463[LOG-3463])
+
+* Before this update, the operator was not available for {product-title} 4.10 clusters because of an incompatibility between {product-title} console and the logging-view-plugin.  With this update, the plugin is properly integrated with the {product-title} 4.10 admin console. (link:https://issues.redhat.com/browse/LOG-3447[LOG-3447])
```
Comment (aminesnow) [inline modules/cluster-logging-rn-5.6.1.adoc:19]: Maybe we should specify which operator we are referencing here

**Comment by aminesnow** [comment]
the Loki related notes LGTM

File: modules/cluster-logging-rn-5.6.1.adoc (hunk 1 above)
Comment (libander) [reply modules/cluster-logging-rn-5.6.1.adoc]: Good point!

**Review by xperimental — APPROVED** [review]

**Comment by anpingli** [comment]
@libander  Shall we include this https://issues.redhat.com/browse/LOG-3441?

File: modules/cluster-logging-rn-5.6.1.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-1_{context}"]
+= Logging 5.6.1
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0634[OpenShift Logging Bug Fix Release 5.6.1].
+
+[id="openshift-logging-5-6-1-bug-fixes"]
+== Bug fixes
+* Before this update, the compactor would report TLS certificate errors from communications with the querier when retention was active. With this update, the compactor or querier no longer communicate erroneously over HTTP. (link:https://issues.redhat.com/browse/LOG-3494[LOG-3494])
```
Comment (gabriel-rh) [inline modules/cluster-logging-rn-5.6.1.adoc:9]:
```suggestion
* Before this update, the compactor would report TLS certificate errors from communications with the querier when retention was active. With this update, the compactor and querier no longer communicate erroneously over HTTP. (link:https://issues.redhat.com/browse/LOG-3494[LOG-3494])
```

File: modules/cluster-logging-rn-5.6.1.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+* Before this update, the Loki Operator would not retry setting the status of the `LokiStack` CR, which caused stale status information. With this update, the Operator retries status information updates on conflict.(link:https://issues.redhat.com/browse/LOG-3496[LOG-3496])
```
Comment (gabriel-rh) [inline modules/cluster-logging-rn-5.6.1.adoc:11]: needs a space before (link:....

**Review by gabriel-rh — COMMENTED** [review]
Looks good - just some very minor suggestions

**Comment by libander** [comment]
> @libander Shall we include this https://issues.redhat.com/browse/LOG-3441?

It is. Line 15.

**Review by anpingli — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-rn-5.6.1.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+* Before this update, the Loki Operator would not retry setting the status of the `LokiStack` CR, which caused stale status information. With this update, the Operator retries status information updates on conflict. (link:https://issues.redhat.com/browse/LOG-3496[LOG-3496])
+
+* Before this update, the Loki Operator Webhook server caused TLS errors when the `kube-apiserver-operator` Operator checked the webhook validity. With this update, the Loki Operator Webhook PKI is managed by the Operator Lifecycle Manager (OLM), resolving the issue. (link:https://issues.redhat.com/browse/LOG-3510[LOG-3510])
+
+* Before this update, the LokiStack Gateway Labels Enforcer generated parsing errors for valid LogQL queries when using combined label filters with boolean expressions. With this update, the LokiStack LogQL implementation supports label filters with boolean expression and resolves the issue. (link:https://issues.redhat.com/browse/LOG-3441[LOG-3441]), (link:https://issues.redhat.com/browse/LOG-3397[LOG-3397])
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.6.1.adoc:15]:
```suggestion
* Before this update, the LokiStack Gateway Labels Enforcer generated parsing errors for valid LogQL queries when using combined label filters with boolean expressions. With this update, the LokiStack LogQL implementation supports label filters with boolean expressions and resolves the issue. (link:https://issues.redhat.com/browse/LOG-3441[LOG-3441]), (link:https://issues.redhat.com/browse/LOG-3397[LOG-3397])
```

**Pull request merged**